### PR TITLE
Use custom text styles instead of text theme.

### PR DESCRIPTION
**Pull request by Thesmader**
Custom text styles are better to handle custom styles for text
throughout the app. Modifying the text theme also changes the style of
the text in the themes that are in use in the material widgets.

### CODE REVIEW

File: elaichi/lib/app/colors.dart [hunk 1]
```diff
@@ -11,9 +11,13 @@ class AppColors {
   static const Color lightBottomNav = Color.fromRGBO(255, 255, 255, 1.0);
   static const Color darkBottomNav = Color.fromRGBO(23, 23, 23, 1.0);
   static const Color lightAnnouncementText = Color.fromRGBO(79, 79, 79, 1.0);
+  static const Color bodyText = Color.fromRGBO(79, 79, 79, 1.0);
+  static const Color titleText = Color.fromRGBO(51, 51, 51, 1.0);
+  static const Color overlineText = Color.fromRGBO(130, 130, 130, 1);
```
Comment (DesignrKnight) [inline elaichi/lib/app/colors.dart:16]: The O values are written as 1.0 and also as 1
I hope both are supported

**Review by DesignrKnight — COMMENTED** [review]
I see that we are using google fonts and also I see a local font. Are both needed?

File: elaichi/lib/app/colors.dart (hunk 1 above)
Comment (Thesmader) [reply elaichi/lib/app/colors.dart:16]: Both are supported. Still making it 1.0

**Comment by Thesmader** [comment]
> I see that we are using google fonts and also I see a local font. Are both needed?

https://github.com/material-foundation/google-fonts-flutter#bundling-font-files-in-your-applications-assets
According to this google_fonts is intelligent enough to recognise the asset fonts and load them in. Google Fonts allows to use diffrenet styles with hot reloading during development. We could maybe get rid of it later.

File: elaichi/lib/app/colors.dart (hunk 1 above)
Comment (Thesmader) [inline elaichi/lib/app/colors.dart:16]:
```suggestion
  static const Color overlineText = Color.fromRGBO(130, 130, 130, 1.0);
```

**Pull request merged**